controlExtensionProject/src/test/java/controlExtension/ControlExtensionPage.java

```diff
@@ -0,0 +1,60 @@
+package controlExtension;
+
+import java.util.List;
+
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+
+public class ControlExtensionPage{
```

Comment:
You shouldn't use this pattern. This has all methods available for every control extensions. In the long run this will become a maintenance problem. Each of these methods belongs in the specific control extension class.